homeassistant/components/axis/device.py

```diff
@@ -16,11 +17,14 @@
 from homeassistant.exceptions import ConfigEntryNotReady
 from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
 from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.util.async_ import safe_wait
 
 from .const import CONF_CAMERA, CONF_EVENTS, CONF_MODEL, DOMAIN, LOGGER
 
 from .errors import AuthenticationRequired, CannotConnect
 
+_LOGGER = logging.getLogger(__name__)
+
```

Comment:
We already import a LOGGER, you can use that instead 👍

Reply:
kk